<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Strict//EN" "http://www.w3.org/TR/xhtml1/DTD/xhtml1-strict.dtd">
<html xmlns="http://www.w3.org/1999/xhtml">
<head><title>R: Empirical Analysis of Digital Gene Expression Data in R</title>
<meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
<link rel="stylesheet" type="text/css" href="R.css" />
</head><body>
<h1> Empirical Analysis of Digital Gene Expression Data in R
<img class="toplogo" src="../../../doc/html/Rlogo.svg" alt="[R logo]" />
</h1>
<hr/>
<div style="text-align: center;">
<a href="../../../doc/html/packages.html"><img class="arrow" src="../../../doc/html/left.jpg" alt="[Up]" /></a>
<a href="../../../doc/html/index.html"><img class="arrow" src="../../../doc/html/up.jpg" alt="[Top]" /></a>
</div><h2>Documentation for package &lsquo;edgeR&rsquo; version 3.18.1</h2>

<ul><li><a href="../DESCRIPTION">DESCRIPTION file</a>.</li>
<li><a href="../doc/index.html">User guides, package vignettes and other documentation.</a></li>
<li><a href="../NEWS">Package NEWS</a>.</li>
</ul>

<h2>Help Pages</h2>


<p style="text-align: center;">
<a href="# "> </a>
<a href="#A">A</a>
<a href="#B">B</a>
<a href="#C">C</a>
<a href="#D">D</a>
<a href="#E">E</a>
<a href="#F">F</a>
<a href="#G">G</a>
<a href="#K">K</a>
<a href="#L">L</a>
<a href="#M">M</a>
<a href="#N">N</a>
<a href="#P">P</a>
<a href="#Q">Q</a>
<a href="#R">R</a>
<a href="#S">S</a>
<a href="#T">T</a>
<a href="#V">V</a>
<a href="#W">W</a>
<a href="#Z">Z</a>
<a href="#misc">misc</a>
</p>

<table width="100%">
<tr><td style="width: 25%;"><a href="edgeR-package.html">edgeR-package</a></td>
<td>Empirical analysis of digital gene expression data in R</td></tr>
</table>

<h2><a name="A">-- A --</a></h2>

<table width="100%">
<tr><td style="width: 25%;"><a href="addPriorCount.html">addPriorCount</a></td>
<td>Add a prior count</td></tr>
<tr><td style="width: 25%;"><a href="adjustedProfileLik.html">adjustedProfileLik</a></td>
<td>Adjusted Profile Likelihood for the Negative Binomial Dispersion Parameter</td></tr>
<tr><td style="width: 25%;"><a href="asdataframe.html">as.data.frame.TopTags</a></td>
<td>Turn a TopTags Object into a Dataframe</td></tr>
<tr><td style="width: 25%;"><a href="makeCompressedMatrix.html">as.matrix.compressedMatrix</a></td>
<td>makeCompressedMatrix</td></tr>
<tr><td style="width: 25%;"><a href="asmatrix.html">as.matrix.DGEList</a></td>
<td>Turn a DGEList Object into a Matrix</td></tr>
<tr><td style="width: 25%;"><a href="aveLogCPM.html">aveLogCPM</a></td>
<td>Average Log Counts Per Million</td></tr>
<tr><td style="width: 25%;"><a href="aveLogCPM.html">aveLogCPM.default</a></td>
<td>Average Log Counts Per Million</td></tr>
<tr><td style="width: 25%;"><a href="aveLogCPM.html">aveLogCPM.DGEGLM</a></td>
<td>Average Log Counts Per Million</td></tr>
<tr><td style="width: 25%;"><a href="aveLogCPM.html">aveLogCPM.DGEList</a></td>
<td>Average Log Counts Per Million</td></tr>
</table>

<h2><a name="B">-- B --</a></h2>

<table width="100%">
<tr><td style="width: 25%;"><a href="meanvar.html">binMeanVar</a></td>
<td>Explore the mean-variance relationship for DGE data</td></tr>
<tr><td style="width: 25%;"><a href="binomTest.html">binomTest</a></td>
<td>Exact Binomial Tests for Comparing Two Digital Libraries</td></tr>
</table>

<h2><a name="C">-- C --</a></h2>

<table width="100%">
<tr><td style="width: 25%;"><a href="calcNormFactors.html">calcNormFactors</a></td>
<td>Calculate Normalization Factors to Align Columns of a Count Matrix</td></tr>
<tr><td style="width: 25%;"><a href="calcNormFactors.html">calcNormFactors.default</a></td>
<td>Calculate Normalization Factors to Align Columns of a Count Matrix</td></tr>
<tr><td style="width: 25%;"><a href="calcNormFactors.html">calcNormFactors.DGEList</a></td>
<td>Calculate Normalization Factors to Align Columns of a Count Matrix</td></tr>
<tr><td style="width: 25%;"><a href="normalizeChIPtoInput.html">calcNormOffsetsforChIP</a></td>
<td>Normalize ChIP-Seq Read Counts to Input and Test for Enrichment</td></tr>
<tr><td style="width: 25%;"><a href="camera.DGEList.html">camera.DGEList</a></td>
<td>Competitive Gene Set Test for Digital Gene Expression Data Accounting for Inter-gene Correlation</td></tr>
<tr><td style="width: 25%;"><a href="commonCondLogLikDerDelta.html">commonCondLogLikDerDelta</a></td>
<td>Conditional Log-Likelihoods in Terms of Delta</td></tr>
<tr><td style="width: 25%;"><a href="makeCompressedMatrix.html">compressedMatrix</a></td>
<td>makeCompressedMatrix</td></tr>
<tr><td style="width: 25%;"><a href="condLogLikDerSize.html">condLogLikDerDelta</a></td>
<td>Conditional Log-Likelihood of the Dispersion for a Single Group of Replicate Libraries</td></tr>
<tr><td style="width: 25%;"><a href="condLogLikDerSize.html">condLogLikDerSize</a></td>
<td>Conditional Log-Likelihood of the Dispersion for a Single Group of Replicate Libraries</td></tr>
<tr><td style="width: 25%;"><a href="cpm.html">cpm</a></td>
<td>Counts per Million or Reads per Kilobase per Million</td></tr>
<tr><td style="width: 25%;"><a href="cpm.html">cpm.default</a></td>
<td>Counts per Million or Reads per Kilobase per Million</td></tr>
<tr><td style="width: 25%;"><a href="cpm.html">cpm.DGEList</a></td>
<td>Counts per Million or Reads per Kilobase per Million</td></tr>
<tr><td style="width: 25%;"><a href="cutWithMinN.html">cutWithMinN</a></td>
<td>Cut numeric vector into non-empty intervals</td></tr>
</table>

<h2><a name="D">-- D --</a></h2>

<table width="100%">
<tr><td style="width: 25%;"><a href="decidetestsDGE.html">decideTests.DGEExact</a></td>
<td>Multiple Testing Across Genes and Contrasts</td></tr>
<tr><td style="width: 25%;"><a href="decidetestsDGE.html">decideTests.DGELRT</a></td>
<td>Multiple Testing Across Genes and Contrasts</td></tr>
<tr><td style="width: 25%;"><a href="decidetestsDGE.html">decideTestsDGE</a></td>
<td>Multiple Testing Across Genes and Contrasts</td></tr>
<tr><td style="width: 25%;"><a href="mglm.html">designAsFactor</a></td>
<td>Fit Negative Binomial Generalized Linear Model to Multiple Response Vectors: Low Level Functions</td></tr>
<tr><td style="width: 25%;"><a href="DGEExact-class.html">DGEExact-class</a></td>
<td>differential expression of Digital Gene Expression data - class</td></tr>
<tr><td style="width: 25%;"><a href="DGEGLM-class.html">DGEGLM-class</a></td>
<td>Digital Gene Expression Generalized Linear Model results - class</td></tr>
<tr><td style="width: 25%;"><a href="DGEList.html">DGEList</a></td>
<td>DGEList Constructor</td></tr>
<tr><td style="width: 25%;"><a href="DGEList-class.html">DGEList-class</a></td>
<td>Digital Gene Expression data - class</td></tr>
<tr><td style="width: 25%;"><a href="DGELRT-class.html">DGELRT-class</a></td>
<td>Digital Gene Expression Likelihood Ratio Test data and results - class</td></tr>
<tr><td style="width: 25%;"><a href="dglmStdResid.html">dglmStdResid</a></td>
<td>Visualize the mean-variance relationship in DGE data using standardized residuals</td></tr>
<tr><td style="width: 25%;"><a href="diffSpliceDGE.html">diffSpliceDGE</a></td>
<td>Test for Differential Exon Usage</td></tr>
<tr><td style="width: 25%;"><a href="dim.html">dim.DGEExact</a></td>
<td>Retrieve the Dimensions of a DGEList, DGEExact, DGEGLM, DGELRT or TopTags Object</td></tr>
<tr><td style="width: 25%;"><a href="dim.html">dim.DGEGLM</a></td>
<td>Retrieve the Dimensions of a DGEList, DGEExact, DGEGLM, DGELRT or TopTags Object</td></tr>
<tr><td style="width: 25%;"><a href="dim.html">dim.DGEList</a></td>
<td>Retrieve the Dimensions of a DGEList, DGEExact, DGEGLM, DGELRT or TopTags Object</td></tr>
<tr><td style="width: 25%;"><a href="dim.html">dim.DGELRT</a></td>
<td>Retrieve the Dimensions of a DGEList, DGEExact, DGEGLM, DGELRT or TopTags Object</td></tr>
<tr><td style="width: 25%;"><a href="dim.html">dim.TopTags</a></td>
<td>Retrieve the Dimensions of a DGEList, DGEExact, DGEGLM, DGELRT or TopTags Object</td></tr>
<tr><td style="width: 25%;"><a href="dimnames.html">dimnames.DGEExact</a></td>
<td>Retrieve the Dimension Names of a DGE Object</td></tr>
<tr><td style="width: 25%;"><a href="dimnames.html">dimnames.DGEGLM</a></td>
<td>Retrieve the Dimension Names of a DGE Object</td></tr>
<tr><td style="width: 25%;"><a href="dimnames.html">dimnames.DGEList</a></td>
<td>Retrieve the Dimension Names of a DGE Object</td></tr>
<tr><td style="width: 25%;"><a href="dimnames.html">dimnames.DGELRT</a></td>
<td>Retrieve the Dimension Names of a DGE Object</td></tr>
<tr><td style="width: 25%;"><a href="dimnames.html">dimnames.TopTags</a></td>
<td>Retrieve the Dimension Names of a DGE Object</td></tr>
<tr><td style="width: 25%;"><a href="dimnames.html">dimnames&lt;-.DGEExact</a></td>
<td>Retrieve the Dimension Names of a DGE Object</td></tr>
<tr><td style="width: 25%;"><a href="dimnames.html">dimnames&lt;-.DGEGLM</a></td>
<td>Retrieve the Dimension Names of a DGE Object</td></tr>
<tr><td style="width: 25%;"><a href="dimnames.html">dimnames&lt;-.DGEList</a></td>
<td>Retrieve the Dimension Names of a DGE Object</td></tr>
<tr><td style="width: 25%;"><a href="dimnames.html">dimnames&lt;-.DGELRT</a></td>
<td>Retrieve the Dimension Names of a DGE Object</td></tr>
<tr><td style="width: 25%;"><a href="dispBinTrend.html">dispBinTrend</a></td>
<td>Estimate Dispersion Trend by Binning for NB GLMs</td></tr>
<tr><td style="width: 25%;"><a href="dispCoxReid.html">dispCoxReid</a></td>
<td>Estimate Common Dispersion for Negative Binomial GLMs</td></tr>
<tr><td style="width: 25%;"><a href="dispCoxReidInterpolateTagwise.html">dispCoxReidInterpolateTagwise</a></td>
<td>Estimate Genewise Dispersion for Negative Binomial GLMs by Cox-Reid Adjusted Profile Likelihood</td></tr>
<tr><td style="width: 25%;"><a href="dispCoxReidSplineTrend.html">dispCoxReidPowerTrend</a></td>
<td>Estimate Dispersion Trend for Negative Binomial GLMs</td></tr>
<tr><td style="width: 25%;"><a href="dispCoxReidSplineTrend.html">dispCoxReidSplineTrend</a></td>
<td>Estimate Dispersion Trend for Negative Binomial GLMs</td></tr>
<tr><td style="width: 25%;"><a href="dispCoxReid.html">dispDeviance</a></td>
<td>Estimate Common Dispersion for Negative Binomial GLMs</td></tr>
<tr><td style="width: 25%;"><a href="dispCoxReid.html">dispPearson</a></td>
<td>Estimate Common Dispersion for Negative Binomial GLMs</td></tr>
<tr><td style="width: 25%;"><a href="dropEmptyLevels.html">dropEmptyLevels</a></td>
<td>Drop Levels of a Factor that Never Occur</td></tr>
</table>

<h2><a name="E">-- E --</a></h2>

<table width="100%">
<tr><td style="width: 25%;"><a href="edgeR-package.html">edgeR</a></td>
<td>Empirical analysis of digital gene expression data in R</td></tr>
<tr><td style="width: 25%;"><a href="edgeRUsersGuide.html">edgeRUsersGuide</a></td>
<td>View edgeR User's Guide</td></tr>
<tr><td style="width: 25%;"><a href="equalizeLibSizes.html">equalizeLibSizes</a></td>
<td>Equalize Library Sizes by Quantile-to-Quantile Normalization</td></tr>
<tr><td style="width: 25%;"><a href="equalizeLibSizes.html">equalizeLibSizes.default</a></td>
<td>Equalize Library Sizes by Quantile-to-Quantile Normalization</td></tr>
<tr><td style="width: 25%;"><a href="equalizeLibSizes.html">equalizeLibSizes.DGEList</a></td>
<td>Equalize Library Sizes by Quantile-to-Quantile Normalization</td></tr>
<tr><td style="width: 25%;"><a href="estimateCommonDisp.html">estimateCommonDisp</a></td>
<td>Estimate Common Negative Binomial Dispersion by Conditional Maximum Likelihood</td></tr>
<tr><td style="width: 25%;"><a href="estimateCommonDisp.html">estimateCommonDisp.default</a></td>
<td>Estimate Common Negative Binomial Dispersion by Conditional Maximum Likelihood</td></tr>
<tr><td style="width: 25%;"><a href="estimateCommonDisp.html">estimateCommonDisp.DGEList</a></td>
<td>Estimate Common Negative Binomial Dispersion by Conditional Maximum Likelihood</td></tr>
<tr><td style="width: 25%;"><a href="estimateDisp.html">estimateDisp</a></td>
<td>Estimate Common, Trended and Tagwise Negative Binomial dispersions by weighted likelihood empirical Bayes</td></tr>
<tr><td style="width: 25%;"><a href="estimateDisp.html">estimateDisp.default</a></td>
<td>Estimate Common, Trended and Tagwise Negative Binomial dispersions by weighted likelihood empirical Bayes</td></tr>
<tr><td style="width: 25%;"><a href="estimateDisp.html">estimateDisp.DGEList</a></td>
<td>Estimate Common, Trended and Tagwise Negative Binomial dispersions by weighted likelihood empirical Bayes</td></tr>
<tr><td style="width: 25%;"><a href="estimateExonGenewisedisp.html">estimateExonGenewiseDisp</a></td>
<td>Estimate Genewise Dispersions from Exon-Level Count Data</td></tr>
<tr><td style="width: 25%;"><a href="estimateGLMCommonDisp.html">estimateGLMCommonDisp</a></td>
<td>Estimate Common Dispersion for Negative Binomial GLMs</td></tr>
<tr><td style="width: 25%;"><a href="estimateGLMCommonDisp.html">estimateGLMCommonDisp.default</a></td>
<td>Estimate Common Dispersion for Negative Binomial GLMs</td></tr>
<tr><td style="width: 25%;"><a href="estimateGLMCommonDisp.html">estimateGLMCommonDisp.DGEList</a></td>
<td>Estimate Common Dispersion for Negative Binomial GLMs</td></tr>
<tr><td style="width: 25%;"><a href="estimateGLMRobustDisp.html">estimateGLMRobustDisp</a></td>
<td>Empirical Robust Bayes Tagwise Dispersions for Negative Binomial GLMs using Observation Weights</td></tr>
<tr><td style="width: 25%;"><a href="estimateGLMTagwiseDisp.html">estimateGLMTagwiseDisp</a></td>
<td>Empirical Bayes Tagwise Dispersions for Negative Binomial GLMs</td></tr>
<tr><td style="width: 25%;"><a href="estimateGLMTagwiseDisp.html">estimateGLMTagwiseDisp.default</a></td>
<td>Empirical Bayes Tagwise Dispersions for Negative Binomial GLMs</td></tr>
<tr><td style="width: 25%;"><a href="estimateGLMTagwiseDisp.html">estimateGLMTagwiseDisp.DGEList</a></td>
<td>Empirical Bayes Tagwise Dispersions for Negative Binomial GLMs</td></tr>
<tr><td style="width: 25%;"><a href="estimateGLMTrendedDisp.html">estimateGLMTrendedDisp</a></td>
<td>Estimate Trended Dispersion for Negative Binomial GLMs</td></tr>
<tr><td style="width: 25%;"><a href="estimateGLMTrendedDisp.html">estimateGLMTrendedDisp.default</a></td>
<td>Estimate Trended Dispersion for Negative Binomial GLMs</td></tr>
<tr><td style="width: 25%;"><a href="estimateGLMTrendedDisp.html">estimateGLMTrendedDisp.DGEList</a></td>
<td>Estimate Trended Dispersion for Negative Binomial GLMs</td></tr>
<tr><td style="width: 25%;"><a href="estimateTagwiseDisp.html">estimateTagwiseDisp</a></td>
<td>Estimate Empirical Bayes Tagwise Dispersion Values</td></tr>
<tr><td style="width: 25%;"><a href="estimateTagwiseDisp.html">estimateTagwiseDisp.default</a></td>
<td>Estimate Empirical Bayes Tagwise Dispersion Values</td></tr>
<tr><td style="width: 25%;"><a href="estimateTagwiseDisp.html">estimateTagwiseDisp.DGEList</a></td>
<td>Estimate Empirical Bayes Tagwise Dispersion Values</td></tr>
<tr><td style="width: 25%;"><a href="estimateTrendedDisp.html">estimateTrendedDisp</a></td>
<td>Estimate Empirical Bayes Trended Dispersion Values</td></tr>
<tr><td style="width: 25%;"><a href="estimateTrendedDisp.html">estimateTrendedDisp.default</a></td>
<td>Estimate Empirical Bayes Trended Dispersion Values</td></tr>
<tr><td style="width: 25%;"><a href="estimateTrendedDisp.html">estimateTrendedDisp.DGEList</a></td>
<td>Estimate Empirical Bayes Trended Dispersion Values</td></tr>
<tr><td style="width: 25%;"><a href="exactTest.html">exactTest</a></td>
<td>Exact Tests for Differences between Two Groups of Negative-Binomial Counts</td></tr>
<tr><td style="width: 25%;"><a href="exactTest.html">exactTestBetaApprox</a></td>
<td>Exact Tests for Differences between Two Groups of Negative-Binomial Counts</td></tr>
<tr><td style="width: 25%;"><a href="exactTest.html">exactTestByDeviance</a></td>
<td>Exact Tests for Differences between Two Groups of Negative-Binomial Counts</td></tr>
<tr><td style="width: 25%;"><a href="exactTest.html">exactTestBySmallP</a></td>
<td>Exact Tests for Differences between Two Groups of Negative-Binomial Counts</td></tr>
<tr><td style="width: 25%;"><a href="exactTest.html">exactTestDoubleTail</a></td>
<td>Exact Tests for Differences between Two Groups of Negative-Binomial Counts</td></tr>
<tr><td style="width: 25%;"><a href="expandAsMatrix.html">expandAsMatrix</a></td>
<td>expandAsMatrix</td></tr>
</table>

<h2><a name="F">-- F --</a></h2>

<table width="100%">
<tr><td style="width: 25%;"><a href="roast.DGEList.html">fry.DGEList</a></td>
<td>Rotation Gene Set Tests for Digital Gene Expression Data</td></tr>
</table>

<h2><a name="G">-- G --</a></h2>

<table width="100%">
<tr><td style="width: 25%;"><a href="getCounts.html">getCounts</a></td>
<td>Extract Specified Component of a DGEList Object</td></tr>
<tr><td style="width: 25%;"><a href="getCounts.html">getDispersion</a></td>
<td>Extract Specified Component of a DGEList Object</td></tr>
<tr><td style="width: 25%;"><a href="dglmStdResid.html">getDispersions</a></td>
<td>Visualize the mean-variance relationship in DGE data using standardized residuals</td></tr>
<tr><td style="width: 25%;"><a href="getCounts.html">getOffset</a></td>
<td>Extract Specified Component of a DGEList Object</td></tr>
<tr><td style="width: 25%;"><a href="getPriorN.html">getPriorN</a></td>
<td>Get a Recommended Value for Prior N from DGEList Object</td></tr>
<tr><td style="width: 25%;"><a href="gini.html">gini</a></td>
<td>Gini dispersion index</td></tr>
<tr><td style="width: 25%;"><a href="glmfit.html">glmFit</a></td>
<td>Genewise Negative Binomial Generalized Linear Models</td></tr>
<tr><td style="width: 25%;"><a href="glmfit.html">glmFit.default</a></td>
<td>Genewise Negative Binomial Generalized Linear Models</td></tr>
<tr><td style="width: 25%;"><a href="glmfit.html">glmFit.DGEList</a></td>
<td>Genewise Negative Binomial Generalized Linear Models</td></tr>
<tr><td style="width: 25%;"><a href="glmfit.html">glmLRT</a></td>
<td>Genewise Negative Binomial Generalized Linear Models</td></tr>
<tr><td style="width: 25%;"><a href="glmQLFTest.html">glmQLFit</a></td>
<td>Genewise Negative Binomial Generalized Linear Models with Quasi-likelihood Tests</td></tr>
<tr><td style="width: 25%;"><a href="glmQLFTest.html">glmQLFit.default</a></td>
<td>Genewise Negative Binomial Generalized Linear Models with Quasi-likelihood Tests</td></tr>
<tr><td style="width: 25%;"><a href="glmQLFTest.html">glmQLFit.DGEList</a></td>
<td>Genewise Negative Binomial Generalized Linear Models with Quasi-likelihood Tests</td></tr>
<tr><td style="width: 25%;"><a href="glmQLFTest.html">glmQLFTest</a></td>
<td>Genewise Negative Binomial Generalized Linear Models with Quasi-likelihood Tests</td></tr>
<tr><td style="width: 25%;"><a href="glmTreat.html">glmTreat</a></td>
<td>Test for Differential Expression Relative to a Threshold</td></tr>
<tr><td style="width: 25%;"><a href="goana.html">goana.DGEExact</a></td>
<td>Gene Ontology or KEGG Analysis of Differentially Expressed Genes</td></tr>
<tr><td style="width: 25%;"><a href="goana.html">goana.DGELRT</a></td>
<td>Gene Ontology or KEGG Analysis of Differentially Expressed Genes</td></tr>
<tr><td style="width: 25%;"><a href="gof.html">gof</a></td>
<td>Goodness of Fit Tests for Multiple GLM Fits</td></tr>
<tr><td style="width: 25%;"><a href="goodTuring.html">goodTuring</a></td>
<td>Good-Turing Frequency Estimation</td></tr>
<tr><td style="width: 25%;"><a href="goodTuring.html">goodTuringPlot</a></td>
<td>Good-Turing Frequency Estimation</td></tr>
<tr><td style="width: 25%;"><a href="goodTuring.html">goodTuringProportions</a></td>
<td>Good-Turing Frequency Estimation</td></tr>
</table>

<h2><a name="K">-- K --</a></h2>

<table width="100%">
<tr><td style="width: 25%;"><a href="goana.html">kegga.DGEExact</a></td>
<td>Gene Ontology or KEGG Analysis of Differentially Expressed Genes</td></tr>
<tr><td style="width: 25%;"><a href="goana.html">kegga.DGELRT</a></td>
<td>Gene Ontology or KEGG Analysis of Differentially Expressed Genes</td></tr>
</table>

<h2><a name="L">-- L --</a></h2>

<table width="100%">
<tr><td style="width: 25%;"><a href="loessByCol.html">locfitByCol</a></td>
<td>Locally Weighted Mean By Column</td></tr>
<tr><td style="width: 25%;"><a href="loessByCol.html">loessByCol</a></td>
<td>Locally Weighted Mean By Column</td></tr>
</table>

<h2><a name="M">-- M --</a></h2>

<table width="100%">
<tr><td style="width: 25%;"><a href="makeCompressedMatrix.html">makeCompressedMatrix</a></td>
<td>makeCompressedMatrix</td></tr>
<tr><td style="width: 25%;"><a href="maPlot.html">maPlot</a></td>
<td>Plots Log-Fold Change versus Log-Concentration (or, M versus A) for Count Data</td></tr>
<tr><td style="width: 25%;"><a href="maximizeInterpolant.html">maximizeInterpolant</a></td>
<td>Maximize a function given a table of values by spline interpolation.</td></tr>
<tr><td style="width: 25%;"><a href="maximizeQuadratic.html">maximizeQuadratic</a></td>
<td>Maximize a function given a table of values by quadratic interpolation.</td></tr>
<tr><td style="width: 25%;"><a href="mglm.html">mglm</a></td>
<td>Fit Negative Binomial Generalized Linear Model to Multiple Response Vectors: Low Level Functions</td></tr>
<tr><td style="width: 25%;"><a href="mglm.html">mglmLevenberg</a></td>
<td>Fit Negative Binomial Generalized Linear Model to Multiple Response Vectors: Low Level Functions</td></tr>
<tr><td style="width: 25%;"><a href="mglm.html">mglmOneGroup</a></td>
<td>Fit Negative Binomial Generalized Linear Model to Multiple Response Vectors: Low Level Functions</td></tr>
<tr><td style="width: 25%;"><a href="mglm.html">mglmOneWay</a></td>
<td>Fit Negative Binomial Generalized Linear Model to Multiple Response Vectors: Low Level Functions</td></tr>
<tr><td style="width: 25%;"><a href="movingAverageByCol.html">movingAverageByCol</a></td>
<td>Moving Average Smoother of Matrix Columns</td></tr>
<tr><td style="width: 25%;"><a href="roast.DGEList.html">mroast.DGEList</a></td>
<td>Rotation Gene Set Tests for Digital Gene Expression Data</td></tr>
</table>

<h2><a name="N">-- N --</a></h2>

<table width="100%">
<tr><td style="width: 25%;"><a href="nbinomDeviance.html">nbinomDeviance</a></td>
<td>Negative Binomial Deviance</td></tr>
<tr><td style="width: 25%;"><a href="nbinomDeviance.html">nbinomUnitDeviance</a></td>
<td>Negative Binomial Deviance</td></tr>
<tr><td style="width: 25%;"><a href="normalizeChIPtoInput.html">normalizeChIPtoInput</a></td>
<td>Normalize ChIP-Seq Read Counts to Input and Test for Enrichment</td></tr>
</table>

<h2><a name="P">-- P --</a></h2>

<table width="100%">
<tr><td style="width: 25%;"><a href="plotBCV.html">plotBCV</a></td>
<td>Plot Biological Coefficient of Variation</td></tr>
<tr><td style="width: 25%;"><a href="plotExonUsage.html">plotExonUsage</a></td>
<td>Create a Plot of Exon Usage from Exon-Level Count Data</td></tr>
<tr><td style="width: 25%;"><a href="plotMD.html">plotMD.DGEExact</a></td>
<td>Mean-Difference Plot of Count Data</td></tr>
<tr><td style="width: 25%;"><a href="plotMD.html">plotMD.DGEGLM</a></td>
<td>Mean-Difference Plot of Count Data</td></tr>
<tr><td style="width: 25%;"><a href="plotMD.html">plotMD.DGEList</a></td>
<td>Mean-Difference Plot of Count Data</td></tr>
<tr><td style="width: 25%;"><a href="plotMD.html">plotMD.DGELRT</a></td>
<td>Mean-Difference Plot of Count Data</td></tr>
<tr><td style="width: 25%;"><a href="plotMDS.DGEList.html">plotMDS.DGEList</a></td>
<td>Multidimensional scaling plot of distances between digital gene expression profiles</td></tr>
<tr><td style="width: 25%;"><a href="meanvar.html">plotMeanVar</a></td>
<td>Explore the mean-variance relationship for DGE data</td></tr>
<tr><td style="width: 25%;"><a href="plotQLDisp.html">plotQLDisp</a></td>
<td>Plot the quasi-likelihood dispersion</td></tr>
<tr><td style="width: 25%;"><a href="plotSmear.html">plotSmear</a></td>
<td>Smear plot</td></tr>
<tr><td style="width: 25%;"><a href="plotSpliceDGE.html">plotSpliceDGE</a></td>
<td>Differential splicing plot</td></tr>
<tr><td style="width: 25%;"><a href="predFC.html">predFC</a></td>
<td>Predictive log-fold changes</td></tr>
<tr><td style="width: 25%;"><a href="predFC.html">predFC.default</a></td>
<td>Predictive log-fold changes</td></tr>
<tr><td style="width: 25%;"><a href="predFC.html">predFC.DGEList</a></td>
<td>Predictive log-fold changes</td></tr>
<tr><td style="width: 25%;"><a href="processAmplicons.html">processAmplicons</a></td>
<td>Process raw data from pooled genetic sequencing screens</td></tr>
</table>

<h2><a name="Q">-- Q --</a></h2>

<table width="100%">
<tr><td style="width: 25%;"><a href="q2qnbinom.html">q2qnbinom</a></td>
<td>Quantile to Quantile Mapping between Negative-Binomial Distributions</td></tr>
<tr><td style="width: 25%;"><a href="q2qnbinom.html">q2qpois</a></td>
<td>Quantile to Quantile Mapping between Negative-Binomial Distributions</td></tr>
</table>

<h2><a name="R">-- R --</a></h2>

<table width="100%">
<tr><td style="width: 25%;"><a href="readDGE.html">readDGE</a></td>
<td>Read and Merge a Set of Files Containing Count Data</td></tr>
<tr><td style="width: 25%;"><a href="roast.DGEList.html">roast.DGEList</a></td>
<td>Rotation Gene Set Tests for Digital Gene Expression Data</td></tr>
<tr><td style="width: 25%;"><a href="romer.DGEList.html">romer.DGEList</a></td>
<td>Rotation Gene Set Tests for Digital Gene Expression Data</td></tr>
<tr><td style="width: 25%;"><a href="cpm.html">rpkm</a></td>
<td>Counts per Million or Reads per Kilobase per Million</td></tr>
<tr><td style="width: 25%;"><a href="cpm.html">rpkm.default</a></td>
<td>Counts per Million or Reads per Kilobase per Million</td></tr>
<tr><td style="width: 25%;"><a href="cpm.html">rpkm.DGEList</a></td>
<td>Counts per Million or Reads per Kilobase per Million</td></tr>
</table>

<h2><a name="S">-- S --</a></h2>

<table width="100%">
<tr><td style="width: 25%;"><a href="scaleOffset.html">scaleOffset</a></td>
<td>Scale offsets</td></tr>
<tr><td style="width: 25%;"><a href="scaleOffset.html">scaleOffset.default</a></td>
<td>Scale offsets</td></tr>
<tr><td style="width: 25%;"><a href="scaleOffset.html">scaleOffset.DGEList</a></td>
<td>Scale offsets</td></tr>
<tr><td style="width: 25%;"><a href="DGEExact-class.html">show-method</a></td>
<td>differential expression of Digital Gene Expression data - class</td></tr>
<tr><td style="width: 25%;"><a href="DGEGLM-class.html">show-method</a></td>
<td>Digital Gene Expression Generalized Linear Model results - class</td></tr>
<tr><td style="width: 25%;"><a href="DGELRT-class.html">show-method</a></td>
<td>Digital Gene Expression Likelihood Ratio Test data and results - class</td></tr>
<tr><td style="width: 25%;"><a href="topTags.html">show-method</a></td>
<td>Table of the Top Differentially Expressed Tags</td></tr>
<tr><td style="width: 25%;"><a href="spliceVariants.html">spliceVariants</a></td>
<td>Identify Genes with Splice Variants</td></tr>
<tr><td style="width: 25%;"><a href="splitIntoGroups.html">splitIntoGroups</a></td>
<td>Split the Counts or Pseudocounts from a DGEList Object According To Group</td></tr>
<tr><td style="width: 25%;"><a href="splitIntoGroups.html">splitIntoGroups.default</a></td>
<td>Split the Counts or Pseudocounts from a DGEList Object According To Group</td></tr>
<tr><td style="width: 25%;"><a href="splitIntoGroups.html">splitIntoGroups.DGEList</a></td>
<td>Split the Counts or Pseudocounts from a DGEList Object According To Group</td></tr>
<tr><td style="width: 25%;"><a href="splitIntoGroups.html">splitIntoGroupsPseudo</a></td>
<td>Split the Counts or Pseudocounts from a DGEList Object According To Group</td></tr>
<tr><td style="width: 25%;"><a href="subsetting.html">subsetting</a></td>
<td>Subset DGEList, DGEGLM, DGEExact and DGELRT Objects</td></tr>
<tr><td style="width: 25%;"><a href="sumTechReps.html">sumTechReps</a></td>
<td>Sum Over Replicate Samples</td></tr>
<tr><td style="width: 25%;"><a href="sumTechReps.html">sumTechReps.default</a></td>
<td>Sum Over Replicate Samples</td></tr>
<tr><td style="width: 25%;"><a href="sumTechReps.html">sumTechReps.DGEList</a></td>
<td>Sum Over Replicate Samples</td></tr>
<tr><td style="width: 25%;"><a href="systematicSubset.html">systematicSubset</a></td>
<td>Take a systematic subset of indices.</td></tr>
</table>

<h2><a name="T">-- T --</a></h2>

<table width="100%">
<tr><td style="width: 25%;"><a href="thinCounts.html">thinCounts</a></td>
<td>Binomial or Multinomial Thinning of Counts</td></tr>
<tr><td style="width: 25%;"><a href="topSpliceDGE.html">topSpliceDGE</a></td>
<td>Top table of differentially spliced genes or exons</td></tr>
<tr><td style="width: 25%;"><a href="topTags.html">topTags</a></td>
<td>Table of the Top Differentially Expressed Tags</td></tr>
<tr><td style="width: 25%;"><a href="topTags.html">TopTags-class</a></td>
<td>Table of the Top Differentially Expressed Tags</td></tr>
</table>

<h2><a name="V">-- V --</a></h2>

<table width="100%">
<tr><td style="width: 25%;"><a href="validDGEList.html">validDGEList</a></td>
<td>Check for Valid DGEList object</td></tr>
</table>

<h2><a name="W">-- W --</a></h2>

<table width="100%">
<tr><td style="width: 25%;"><a href="weightedCondLogLikDerDelta.html">weightedCondLogLikDerDelta</a></td>
<td>Weighted Conditional Log-Likelihood in Terms of Delta</td></tr>
<tr><td style="width: 25%;"><a href="WLEB.html">WLEB</a></td>
<td>Calculate Weighted Likelihood Empirical Bayes Estimates</td></tr>
</table>

<h2><a name="Z">-- Z --</a></h2>

<table width="100%">
<tr><td style="width: 25%;"><a href="zscoreNBinom.html">zscoreNBinom</a></td>
<td>Z-score Equivalents of Negative Binomial Deviate</td></tr>
</table>

<h2><a name="misc">-- misc --</a></h2>

<table width="100%">
<tr><td style="width: 25%;"><a href="makeCompressedMatrix.html">[.compressedMatrix</a></td>
<td>makeCompressedMatrix</td></tr>
<tr><td style="width: 25%;"><a href="subsetting.html">[.DGEExact</a></td>
<td>Subset DGEList, DGEGLM, DGEExact and DGELRT Objects</td></tr>
<tr><td style="width: 25%;"><a href="subsetting.html">[.DGEGLM</a></td>
<td>Subset DGEList, DGEGLM, DGEExact and DGELRT Objects</td></tr>
<tr><td style="width: 25%;"><a href="subsetting.html">[.DGEList</a></td>
<td>Subset DGEList, DGEGLM, DGEExact and DGELRT Objects</td></tr>
<tr><td style="width: 25%;"><a href="subsetting.html">[.DGELRT</a></td>
<td>Subset DGEList, DGEGLM, DGEExact and DGELRT Objects</td></tr>
<tr><td style="width: 25%;"><a href="subsetting.html">[.TopTags</a></td>
<td>Subset DGEList, DGEGLM, DGEExact and DGELRT Objects</td></tr>
</table>
</body></html>
